index.html

```diff
@@ -1,11 +1,757 @@
 <!DOCTYPE html>
-  <html>
-      <head>
-        <title>NY Times Clone</title>
-        <link href="styles.css" rel="stylesheet" type="text/css">
-        <meta charset="utf-8">
-      </head>
-      <body>
-        
-      </body>
-  </html>
\ No newline at end of file
+<html>
+  <head>
+    <title>NY Times Clone</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <meta charset="utf-8">
+  </head>
+  <body>
+    <main>
+      <div class="container">
+        <header>
+          <div class="box-icons">
+            <div class="header-icon">
+              <img
+                src="https://img.icons8.com/ios-filled/50/000000/menu.png"
+                alt="hamburger icon">
+            </div>
+            <div class="header-icon">
+              <img
+                src="https://img.icons8.com/material/24/000000/zoom-out.png"
+                alt="search">
+            </div>
+            <div>
+              <span class="spc font">SPACE & COSMOS</span>
+            </div>
+          </div>
+          <div class="logo-container">
+            <img src="Images/NY-logo.png" class="ny-logo" alt="NY-logo">
+          </div>
+          <div class="button-container">
+            <div class="button-div">
+              <button>LOG IN</button>
+            </div>
+            <div class="button-div">
+              <button>SUBSCRIBE NOW</button>
+            </div>
+          </div>
+        </header>
+      </div>
+      <section class="first-sctn">
+        <div class="article-title font">
+          <h1>Space Ripples Reveal</h1>
+          <br />
+          <h1>Big Bang’s Smoking Gun</h1>
+        </div>
+        <div class="div-cover">
+          <img
```

Comment:
`REQUIRED`

Images and videos should be enclosed by a `<figure>` tag with `<figcaption>` that gives a description of the embedded image or video